RevenueCatUI/Data/PaywallViewConfiguration.swift

```diff
@@ -70,16 +70,18 @@ extension PaywallViewConfiguration {
         fonts: PaywallFontProvider = DefaultPaywallFontProvider(),
         displayCloseButton: Bool = false,
         introEligibility: TrialOrIntroEligibilityChecker? = nil,
-        purchaseHandler: PurchaseHandler? = nil
+        purchaseHandler: PurchaseHandler? = PurchaseHandler.default()
     ) {
+        let handler = purchaseHandler ?? PurchaseHandler.default()
```

Comment:
is this needed now that there's a default value in the parameters?

Reply:
No, it's not! The function parameter doesn't need to be optional anymore either.